lib/http/preprocessors.js

```diff
@@ -68,6 +77,8 @@ const authHandler = ({ Sessions, Users, Auth, bcrypt }, context) => {
 
   // Basic Auth, which is allowed over HTTPS only:
   } else if (isPresent(authHeader) && authHeader.startsWith('Basic ')) {
+    // REVIEW: do web users still have legitimate reasons for accessing using BasicAuth?
```

Comment:
Basic auth is needed for generic OData consumers like PowerBI and Excel, unfortunately. I'm not sure there's a way to address that other than something like an API key in URL.